src/server.cpp

```diff
@@ -2722,6 +2767,82 @@ void Server::UpdateCrafting(Player* player)
 	plist->changeItem(0, preview);
 }
 
+bool Server::handleChat(const std::string &name, const std::wstring &wname,
+	const std::wstring &wmessage, std::wstring &answer_to_sender,
+	u16 peer_id_to_avoid_sending)
+{
+	// If something goes wrong, this player is to blame
+	RollbackScopeActor rollback_scope(m_rollback,
+		std::string("player:") + name);
+
+	// Line to send
+	std::wstring line;
+	// Whether to send to the player that sent the line, or to all players
+	bool send_to_sender_only = false;
```

Comment:
Maybe this variable name would be more concise as: "is_message_broadcast" or similar.